### DISCUSSION TITLE
Make Ralph functionalities more obvious to new users

### DISCUSSION THREAD

**Discussion by Leobouloc** [openfun/ralph#429] [Ideas]
When I first discovered Ralph, it took me some digging into the code and discussions with team members to fully understand the multiple facets of Ralph. In particular, the data processing/validating tool AND the LRS server. I believe we would benefit from improving the doc and code structure to make clearer these different aspects. 

Below is a first suggestion. Perhaps we could discuss here of other ways to make Ralph more accessible to first time users ?

## Separate data processing and LRS management in cli ?

Currently, the `ralph` command is used both to:
- read / write / transform statements 
- manage the Ralph LRS by creating users (by writing to `auth.json`)

This seems like very different tasks. I am wondering if it would make things clearer for new users to separate these functions by having two commands. For example `ralph` and `ralph-LRS-admin` ?

**Comment by jmaupetit** [comment]
Moving this issue to a new discussion!

As it's a lot of work to maintain a CLI, I would prefer only have a single `ralph` command. But, I think you are right that it needs some clarification.

Maybe having an `lrs` sub-command would mitigate the situation:

```sh
# the former `ralph runserver` command would become:
ralph lrs run

# the former `ralph auth` command would become:
ralph lrs auth
```

WDYT?

**Comment by Leobouloc** [comment]
Continuing the conversation on clarity of the documentation. 

- Currently the landing page for the documentation lists 3 main functionalities (LRS, CLI, and Library). Unless I am mistaken, CLI and library cover the same functionalities (data fetch, push, validate, convert) but are simply different interfaces. I think it would make sense to describe Ralph as having 2 functionalities (LRS and ETL tools for example);

- Additionally, the current documentation lists 3 "Features" but which are not the the same as the 3 functionalities (except for LRS). I think the "Features" section should at least include the 3 functionalities indicated in the landing page (in the same terms to be less confusing).
